modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -42,16 +42,17 @@ type PolicyDetailsType = {
 	year: string;
 };
 
+enum NavBarLabel {
+	Drivers = 'Drivers',
+	Vehicles = 'Vehicles',
+}
+
 const PolicyDetail = ({
 	dataJSON,
 	email,
 	phone,
-}: ApplicationPolicyDetailsType) => {
-	enum navBarLabels {
-		Vehicles = 'Vehicles',
-		Drives = 'Drives',
-	}
-	const navbarLabel = [navBarLabels.Vehicles, navBarLabels.Drives];
+}: ApplicationPolicyDetailsType) => {	
+	const navbarLabel = [NavBarLabel.Drivers, NavBarLabel.Vehicles];
```

Comment:
The order here matters, because it's an array, it may cause an unexpected behavior somewhere

```suggestion
	const navbarLabel = [NavBarLabel.Vehicles, NavBarLabel.Drivers];
```